app/init.php

```diff
@@ -135,6 +135,8 @@
 // Response related
 const MAX_OUTPUT_CHUNK_SIZE = 2*1024*1024; // 2MB
 
+Cache::setCaseSensitivity(false);
```

Comment:
I had a similar comment in here 
https://github.com/utopia-php/database/pull/117

If majority of use cases require the caseSensitivity to be turned off, lets keep false as the default in 
https://github.com/utopia-php/cache/blob/02ffbe165f9632e05519eb75a2ce32deb19638a0/src/Cache/Cache.php#L15